scipy/stats/_continuous_distns.py

```diff
@@ -11148,7 +11146,8 @@ def _single_cdf(q, k, df):
         ufunc = np.frompyfunc(_single_cdf, 3, 1)
 
         # clip p-values to ensure they are in [0, 1].
-        return np.clip(np.float64(ufunc(x, k, df)), 0, 1)
+        res = np.asarray(ufunc(x, k, df), dtype=np.float64)
+        return np.clip(res, 0, 1)
```

Comment:
```suggestion
        return np.clip(np.float64(ufunc(x, k, df)), 0, 1)
```
I don't want to do this here.